root/types/wsjs.js

```diff
@@ -9,7 +9,7 @@
 
 declare type WsJsRelationshipEntityT =
   | {
-      +entityType: NonUrlCoreEntityTypeT,
+      +entityType: NonUrlRelatableEntityTypeT,
       +gid: string,
       +name: string,
     }
```

Comment:
There is a resolved (?) conversation started by @reosarevok in https://github.com/metabrainz/musicbrainz-server/commit/91b0577b7cb3869b9f69ee2c77be4281f02b3ca2#r103559074 about this part of commit https://github.com/metabrainz/musicbrainz-server/pull/2896/commits/8d530b8a7b2ea0781dbbc6479b85d366bd63be35 by @mwiencek.